pkg/storegateway/bucket.go

```diff
@@ -773,41 +763,6 @@ func storeCachedSeries(ctx context.Context, indexCache indexcache.IndexCache, us
 	indexCache.StoreSeries(ctx, userID, blockID, entry.MatchersKey, shard, data)
 }
 
-// debugFoundBlockSetOverview logs on debug level what exactly blocks we used for query in terms of
-// labels and resolution. This is important because we allow mixed resolution results, so it is quite crucial
-// to be aware what exactly resolution we see on query.
-// TODO(bplotka): Consider adding resolution label to all results to propagate that info to UI and Query API.
-func debugFoundBlockSetOverview(logger log.Logger, mint, maxt, maxResolutionMillis int64, bs []*bucketBlock) {
```

Comment:
Note to reviewers: we never used this in Mimir, since it was made to debug resolutions.